source/emitSchema.ts

```diff
@@ -5,85 +5,109 @@ import type {
   SchemaProperties,
   TypedSchemaProperty,
 } from "@ftrack/api";
+import { type TypeScriptEmitter } from "./typescriptEmitter";
+import { isSchemaTypedContext } from "./utils";
 
-let errors: string[] = [];
 // Add schemas from the schemas folder, to be used for finding extended schemas
-export async function convertSchemaToInterface(
+export async function emitSchemaInterface(
+  typescriptEmitter: TypeScriptEmitter,
   schema: Schema,
   allSchemas: QuerySchemasResponse
 ) {
-  let interfaceName = getId(schema);
+  const interfaceName = getTypeScriptInterfaceNameForInterface(schema);
+
   // If the schema is a subtype of TypedContext, return that
   if (typeof schema?.alias_for === "object" && schema.alias_for.id === "Task") {
-    return {
-      TSInterface: `export type ${interfaceName} = TypedContext<"${interfaceName}">;`,
-      conversionErrors: [],
-    };
+    typescriptEmitter.appendCode(`
+      type ${interfaceName} = TypedContextForSubtype<"${interfaceName}">;
```

Comment:
Not exporting this means you can't use them? Why was this done?

Reply:
When I manually add export to them, everything seems to work great again

Reply:
Yeah this was a pretty serious oversight - embarassing. I have fixed it now. Thank you.